doc.go

```diff
@@ -126,6 +126,8 @@ The following connection parameters are supported:
   - clientConfigFile: specifies the location of the client configuration json file.
     In this file you can configure Easy Logging feature.
 
+  - disableConsoleLogin: true by default. Set to true false to disable console login.
```

Comment:
Updated doc.go. Please check. Thanks!

Reply:
The description isn't correct because we support multiple SAML regardless disableConsoleLogin. As it is internal parameter, I suggest removing it completely from the doc. 
@sfc-gh-pfus WDYT?

Reply:
Yeah, I agree with Darek. IMHO we don't need this parameter documented for now.

Reply:
Removed the parameter description in doc.go